README.md

```diff
@@ -135,6 +135,35 @@ $ python aecs.py
 }
 ```
 
+#### Evaluation of the k-anonymity
```

Comment:
I think "Computing the k-anonymity" is more precise because "Evaluation of the k-anonymity" means analyzing the properties of k-anonymity.